tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -552,6 +552,12 @@ test_i2c_bmp388: bmp388@55 {
 	int-gpios = <&test_gpio 0 0>;
 };
 
+test_i2c_bmp581: bmp581@46 {
```

Comment:
I see that tests are failing due to duplicated address here. 46 is also being used by tmp112. It might be best to move bmp581 entry in dts to down to the end of the devices here.

Reply:
The test was also failing because of a missing `CONFIG_BMP581` in the tests prj.conf.

Reply:
@talhaHavadar is right, the convention is to add new device at the end of the file with the next sequential address. Note this is just a build test and the address does not need to match the sensor datasheet.

Also, you should not need to set `CONFIG_BMP581` in `prj.conf` if you configure the Kconfig symbol to `default y`

Reply:
:ballot_box_with_check: